### PR TITLE
Add support to trigger pipelines for tags

### PR DESCRIPTION
**Pull request by darkLord19**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,21 +9,28 @@ This plugin uses the CircleCI Orb for Mattermost Plugin by **[@nathanaelhoun](ht
...
--   [Slash Commands](#slash-commands)
-    -   [Overview](#overview)
-    -   [Subscribe to webhooks notifications](#subscribe-to-webhooks-notifications)
--   [Frequently Asked Questions](#frequently-asked-questions)
--   [Development](#development)
--   [License](#license)
--   [Contributing](#contributing)
--   [Thanks to](#thanks-to)
--   [Security Vulnerability Disclosure](#security-vulnerability-disclosure)
+- [Mattermost Plugin CircleCI](#mattermost-plugin-circleci)
```
Comment (darkLord19) [inline README.md]: @nathanaelhoun my editor keeps auto formatting it to this based on hierarchy. do you prefer this one or earlier version?

Comment (nathanaelhoun) [reply README.md]: I prefer the simplified one. I think having all the subtitles in the toc make it seems complicated

Comment (nathanaelhoun) [reply README.md]: And IMO having the title of the doc and `Table of Contents` in the table of content provide no value

**Review by nathanaelhoun — APPROVED** [review]
LGTM! :tada:

**Pull request merged**